vispy/plot/plotwidget.py

```diff
@@ -236,7 +236,8 @@ def mesh(self, vertices=None, faces=None, vertex_colors=None,
         meshdata : MeshData | None
             Meshdata to use. If not None, then vertices, faces, and fname
             must be None.
-
+        shading : str | None
+            Shading to use.
         Returns
```

Comment:
@ktritz Regarding the docstring violation, there might be an empty line missing before `Returns`.